common/static/common/templates/discussion/forum-action-answer.underscore

```diff
@@ -5,6 +5,6 @@
             <span class="label-unchecked"><%- gettext("Mark as Answer") %></span>
             <span class="label-checked"><%- gettext("Unmark as Answer") %></span>
         </span>
-        <span class="action-icon"><i class="icon fa fa-ok"></i></span>
+        <span class="action-icon"><i class="icon fa fa-check"></i></span>
```

Comment:
Context: The first two files are changed to fix a visual regression that caused the "endorse" and "mark as answer" icons to have no icon.

Screenshot:
![image](https://cloud.githubusercontent.com/assets/2023680/9751723/43fe96f0-5676-11e5-8ca5-8339a5eeefb5.png)